client/www/js/controllers/leftNavController.js

```diff
@@ -18,8 +18,11 @@ angular.module('amblr.leftnav', ['ngDraggable'])
     $scope.isLoggedIn();
   };
   
-  $scope.$on('draggable:end', function(target, event) {
-    // hardcoded offset to 27 px
-    $scope.$broadcast('newMarkerDrop', event.x, event.y - 27);
+  $scope.$on('draggable:end', function (target, event) {
+    /* only trigger for dropMarker draggable:end event */
+    if (event.data && event.data.type === 'dropMarker') {
```

Comment:
Added to prevent this from triggering on draggable:end when dragging POIs on the route management modal.